### PR TITLE
cdecl: add new port for cdecl

### PR DESCRIPTION
**Pull request by joelburton**
Add new port for cdecl, a console tool for C/C++ declaration/cast interpretation.

###### Tested on
macOS 10.12.2

###### Verification 
Have you
- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [n/a] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [n/a] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/cdecl/Portfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+name                cdecl
+version             3.1.3
+categories          devel
+platforms           darwin
+maintainers         joelburton.com:joel
+description         Tool to compose or decipher C/C++ type declarations and casts.
+long_description    This is a command-line tool to compose or decipher C and C++ \
+                    type declarations and casts.
+homepage            https://github.com/paul-j-lucas/cdecl
+master_sites        https://github.com/paul-j-lucas/cdecl/releases/download/cdecl-3.1.3/
```
Comment (mojca) [inline devel/cdecl/Portfile]: It would probably be nice to use the `github` PortGroup.

File: devel/cdecl/Portfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+name                cdecl
+version             3.1.3
+categories          devel
+platforms           darwin
+maintainers         joelburton.com:joel
```
Comment (mojca) [inline devel/cdecl/Portfile]: Please add you github handle
```
maintainers         {joelburton.com:joel @joelburton}
```
and potentially a keyword `openmaintainer` if you are ok with other developers doing minor changes to the port.

**Comment by joelburton** [comment]
- Switched to GitHub port group
- Added my GH handle
- Add openmaintener

**Comment by l2dy** [comment]
Fixed in 50ff3cef127d0976f4ff7235a0aee66db45cef0d.

**Pull request closed without merging**